charter.md

```diff
@@ -53,15 +53,165 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members.
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected steering committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible. [Eligible voters][voter-eligibility] from the most
+recent election will vote in the special election i.e., eligibility will not be
+redetermined at the time of the special election. A committee member elected in
+a special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
```

Comment:
Good point, I think this case it would make sense to consider this an abstention by default. Lang can be updated via the work done in https://github.com/kubernetes/steering/issues/250 ?

Reply:
I was working with the assumption that unanimous meant everyone _but_ the person being potentially removed. 

💯 agree this needs clarification so that we can clearly understand the intentions and voting requirements when there are 4 members so that it's not left to presumptions and interpretations. 👍

Reply:
I agree, thank you.

Reply:
That works!